### DISCUSSION TITLE
What's happened to pixls.us?

### DISCUSSION THREAD

**Discussion by OldNick** [Beep6581/RawTherapee#6797] [General]
Does anyone know what's happened to pixls.us - and by extension our forum?

I've been getting "unable to connect" for nearly a week and from a site (forget which) I'm told it's not me it's down for everyone.

**Comment by mbaz** [comment]
It seems the server had a hardware failure and it's taking some time to bring it up again. From the `#discuss` channel on Matrix:

https://matrix.to/#/!UOhxnQqDxgwaFKTsNm:matrix.org/$ZxoC01IFpoyKv2vQtxPERnbSnX2JaZlhxzW5SZvJkvw?via=matrix.org

**Comment by Benitoite** [comment]
ref. https://github.com/Beep6581/RawTherapee/issues/6792
